proj.sh

```diff
@@ -6,7 +6,8 @@ function generate-data {
   echo 'Data generation started.'
   source .env && yarn --cwd $DATA_GENERATOR_PATH start $DATA_FILES_PATH "$@"
   /bin/cp -rf $DATA_GENERATOR_PATH/data/* neo4j/import
-  /bin/cp  -f $DATA_GENERATOR_PATH/data/hpaRna.json api/src/data/
+  /bin/cp  -f $DATA_GENERATOR_PATH/data/hpaRna.json api/src/data/ # this line should be removed after HPA data is included in the dataOverlay folder
+  /bin/cp -rf $DATA_GENERATOR_PATH/data/dataOverlay api/
```

Comment:
I have made a comment about this in `neo4j-data-generation`, suggesting:

```suggestion
  /bin/cp -rf $DATA_GENERATOR_PATH/dataOverlay api/
```

Reply:
Looks good. Leaving this as pending until the other related discussions and possible changes are resolved.